cmds/utk/utk.go

```diff
@@ -56,23 +56,62 @@ package main
 import (
 	"flag"
 	"fmt"
+	"strconv"
+	"strings"
 
 	"github.com/linuxboot/fiano/pkg/log"
+	"github.com/linuxboot/fiano/pkg/uefi"
 	"github.com/linuxboot/fiano/pkg/utk"
 	"github.com/linuxboot/fiano/pkg/visitors"
 )
 
-func init() {
+type config struct {
+	ErasePolarity *byte
+}
+
+func parseArguments() (config, []string, error) {
 	flag.Usage = func() {
 		fmt.Fprintf(flag.CommandLine.Output(), "Usage: utk [flags] <file name> [0 or more operations]\n")
 		flag.PrintDefaults()
 		fmt.Fprintf(flag.CommandLine.Output(), "\nOperations:\n%s", visitors.ListCLI())
 	}
+	erasePolarityFlag := flag.String("erase-polarity", "", "set erase polarity; possible values: '', '0x00', '0xFF'")
+	flag.Parse()
+
+	var cfg config
+
+	switch {
+	case *erasePolarityFlag == "":
+	case strings.HasPrefix(*erasePolarityFlag, "0x"):
```

Comment:
You don't need to do this. If you do strconv.ParseUint with a base of 0, it will detect the base automatically and include support for stuff like Octal (not that I'm sure anyone should attempt to supply it in Octal  :P)

Reply:
Wow. Didn't know that. Thanks, will fix!

Reply:
Fixed.